invenio_app_rdm/theme/views.py

```diff
@@ -41,6 +41,12 @@ def init_menu():
             order=1
         )
 
+        current_menu.submenu('requests.deposit').register(
+            'invenio_app_rdm_records.dashboard_requests',
+            _('My dashboard'),
```

Comment:
You can use here the `invenio_app_rdm_records.dashboard` endpoint and pass this argument https://github.com/inveniosoftware/flask-menu/blob/master/flask_menu/__init__.py#L112  i.e 
```
endpoint_arguments_constructor=lamda _: dict(route="/requests")
```